### ISSUE TITLE
chd values for vertex grid

### ISSUE THREAD

**Issue by tomvansteijn** [ArtesiaWater/nlmod#92]
The `chd` function in `gwf.py` does not actually take values from the variable in the dataset ds in the case of a vertex grid:

```
    elif ds.gridtype == "vertex":
        cellids = np.where(ds[chd])
        chd_rec = list(zip(zip(cellids[0], cellids[1]), [1.0] * len(cellids[0])))
```

I think something is missing in this part of the code.

**Comment by dbrakenhoff** [comment]
Thanks for submitting this issue, we'll get working on fixing this.

**Comment by dbrakenhoff** [comment]
Fixed in dev branch by @rubencalje in 76c140a4f1510e778fcba34270fca29352c17ae7, see here:
https://github.com/ArtesiaWater/nlmod/blob/9c7b55cab0dd4937a62abce7201287756929c63c/nlmod/gwf/gwf.py#L414-L426
